GeeksCoreLibrary/Core/Cms/CmsSettings.cs

```diff
@@ -135,6 +136,21 @@ public class CmsSettings
         )]
         public int CacheMinutes { get; set; }
 
+        /// <summary>
+        /// The regular expression for generating the unique cache key or file name.
+        /// </summary>
+        [CmsProperty(
+            PrettyName = "Cache regex",
+            Description = @"<p>The regular expression to use for deciding the unique cache key. The value of each named group will be added to the cache key or file name. This is useful for a main menu for example, to cache the main menu separately for each main category on the site, so that the selected item in the menu will always be correct, even with caching.</p>
+                            <p>Example: If you use the regex ""\/(?<page>[^\/]*)?(\/.*)?"". This regex has 3 named groups (category, subCategory and product). If the user would then open the URL ""/products/drinks/soda/coca-cola/"", then the key or file name for the cache would be ""dynamicContent_123_drinks_soda_coca-cola"". This way the component will be cached separately for each product on the website.</p>
```

Comment:
Volgens mij klopt de uitleg hier niet. Er staat hier dat de regex 3 named groups heeft, maar ik zie er maar 1. En ik weet niet of je "Coca Cola" hier wel moet benoemen aangezien dat een geregistreerd merk is.